engine/dlib/src/dlib/dstrings.cpp

```diff
@@ -193,7 +193,7 @@ size_t dmStrlCpy(char *dst, const char *src, size_t siz)
  * Appends src to string dst of size siz (unlike strncat, siz is the
  * full size of dst, not space left).  At most siz-1 characters
  * will be copied.  Always NUL terminates (unless siz == 0).
- * Returns strlen(src); if retval >= siz, truncation occurred.
+ * Returns strlen(dst) + strlen(src); if retval >= siz, truncation occurred.
```

Comment:
WHy isn't this change in the other PR, that actually alters the bevhavior in dstrings.cpp?